### PR TITLE
CI setup: coverage, testing catalog

### PR DESCRIPTION
**Pull request by nl0**
Test catalog on travis, upload code coverage stats.

### CODE REVIEW

**Comment by nl0** [comment]
@akarve i've put this change into separate pr, so we can see how coverage changes by every subsequent pr. rtm

File: .travis.yml [hunk 1]
```diff
@@ -22,33 +26,64 @@ matrix:
...
+  - |
+    if [[ "$TEST_CATALOG" = true ]]; then
+      cd catalog && npm ci
+    fi
 
 script:
-  - pytest compiler
+  - |
+    if [[ "$TEST_COMPILER" = true ]]; then
+      pytest --cov=./ compiler
```
Comment (meffij) [inline .travis.yml]: should this be `./compiler`? the registry case uses `./registry`

Comment (nl0) [reply .travis.yml]: idk, it was `- pytest compiler` for quite some time (see diff and history for details), and i haven't touched it

Comment (meffij) [reply .travis.yml]: OK sounds good

**Pull request merged**